docs/root/configuration/http/http_filters/aws_lambda_filter.rst

```diff
@@ -0,0 +1,132 @@
+
+.. _config_http_filters_aws_lambda:
+
+AWS Lambda
+==========
+
+* :ref:`v2 API reference <envoy_api_msg_config.filter.http.aws_lambda.v2alpha.config>`
+* This filter should be configured with the name *envoy.filters.http.aws_lambda*.
+
+.. attention::
+
+  The AWS Lambda filter is currently under active development.
+
+The HTTP AWS Lambda filter is used to trigger an AWS Lambda function from a standard HTTP/1.x or HTTP/2 request.
+It supports a few options to control whether to pass through the HTTP request payload as is or to wrap it in a JSON
+schema.
+
+If ``payload_passthrough`` is set to ``true``, then the payload is sent to Lambda without any transformations.
+*Note*: This means you lose access to all the HTTP headers in the Lambda function.
+
+However, if ``payload_passthrough`` is set to ``false``, then the HTTP request is transformed to a JSON payload with the following
```

Comment:
This is not implemented yet, right? I'm a little confused about what is implemented right now versus what is not. Should you just remove the docs from this PR until it's actually implemented?

Reply:
I added it to get a review on the examples mainly. I can remove the parts of the documentation (and examples) that speak to features that are not implemented yet.

If you prefer to review documentation once at the end, that works too. LMK.